PluralKit.Bot/Commands/SystemEdit.cs

```diff
@@ -232,8 +233,26 @@ async Task ShowIcon()
         
         public async Task SystemProxy(Context ctx)
         {
-            ctx.CheckSystem().CheckGuildContext();
-            var gs = await _db.Execute(c => _repo.GetSystemGuild(c, ctx.Guild.Id, ctx.System.Id));
+            ctx.CheckSystem();
+            string word = ctx.PeekArgument() ?? throw new PKSyntaxError("You must pass either \"on\" or \"off\".");
```

Comment:
This "match guild" code should probably be in `ContextEntityArgumentsExt`, similarly to "MatchChannel"